src/optimizer/query_to_operator_transformer.cpp

```diff
@@ -584,9 +584,19 @@ void QueryToOperatorTransformer::Visit(parser::CopyStatement *op, parser::ParseR
   }
 }
 
-void QueryToOperatorTransformer::Visit(UNUSED_ATTRIBUTE parser::AnalyzeStatement *op,
+void QueryToOperatorTransformer::Visit(parser::AnalyzeStatement *op,
                                        UNUSED_ATTRIBUTE parser::ParseResult *parse_result) {
   OPTIMIZER_LOG_DEBUG("Transforming AnalyzeStatement to operators ...");
+  std::vector<catalog::col_oid_t> columns;
+  auto tb_oid = accessor_->GetTableOid(op->GetAnalyzeTable()->GetTableName());
```

Comment:
If there is a namespace specified, shouldn't this instead use:
`CatalogAccessor::GetTableOid(namespace_oid_t ns, std::string name)`

Reply:
A similar thing is done in create trigger. Does it make a difference?

Reply:
Now there are no more namespace id